flocker/node/_deploy.py

```diff
@@ -23,18 +23,18 @@
 from ..common import gather_deferreds
 
 
-def _to_volume_name(name):
+def _to_volume_name(dataset_id):
```

Comment:
A number of callers of `_to_volume_name` remain in this file which pass `self.volume.name` where `self.volume` is an `AttachedVolume` instance.  `AttachedVolume.name` is still documented as a short, human-readable identifier.